### ISSUE TITLE
BEAR navigation burger menu

### ISSUE THREAD

**Issue by iteles** [dwyl/bestevidence#404]
As a user who is creating a BEAR
I would like a way to quickly access any part of the BEAR process
So that I can reach them quickly and easily without having to click back and forth through each and every section

![screen shot 2018-08-09 at 21 47 18](https://user-images.githubusercontent.com/4185328/43926563-a396d056-9c22-11e8-833d-c8ca58a9a291.png)

## Acceptance Criteria
+ [x] The BEAR menu exists as a burger menu in the top right corner of the page (we need to ask @sophielevens how she had envisioned this for desktop!)
+ [x] Clicking on this menu occupies the full screen on mobile
+ [x] There is a `X` at the top - when clicked, this closes the menu
+ [x] It includes a `Create BEAR` title
+ [x] It includes links to each of the BEAR pages (note: some titles have changed and 'Create PDF' is not in scope for this sprint so must be excluded #403)
  + [x] Clicking on these links takes the user to the corresponding page
+ [x] We need to test that going back through the process or editing any one of these page overwrites the data correctly

**Comment by RobStallion** [comment]
After beginning work on the above I realised that this would not work with how the BEAR flow currently works. The reason this will not work is because multiple papers can be appraised from a single `PICO search`. 

What this means is that if a user does a pico search they could have multiple `paper details`/`check validity`/calculate results`/`relevance` sections.

INSERT PICUTRE HERE - will upload shortly

This means that a menu would have no way of knowing which `paper details` page that it needs to go to as the app currently stands. 

Building this functionality in is possible but would require the UI to be changed in some way which would allow a user to see a list of their previous BEARS and to be able to edit each one individually, regardless of if it originated from the same search term as another bear. Also the `choose your paper` section will not allow a user to select a new paper. It will either not exist in the menu or it will only show the one paper that you selected (probably will go with option 1)

After speaking with @ajburls about this we have decided that it will not fit in the scope of this sprint. We have decided to put in a menu that will only direct a user between the last 4 pages of the menu for the time being (e.g. `paper details`/`check validity`/calculate results`/`relevance`).

@ajburls has asked if we can time estimate how long it will take to build in the new functionality and redesign the UI where needed to improve user flow with BEARS.
